less/paperhive/document.less

```diff
@@ -9,6 +9,14 @@
   }
 }
 
+.ph-document-journal {
+  max-width: 250px;
+  overflow: hidden;
+  display: inline-block;
+  text-overflow: ellipsis;
+  vertical-align: top;
+}
```

Comment:
Thanks @pure180. I tested the code on Sauce Labs and it worked very well for FF, Chrome, Safari, and Edge. So I think that should be sufficient. ;-)